packages/riverpod/lib/src/devtool.dart

```diff
@@ -70,13 +117,26 @@ class RiverpodBinding {
     return binding!;
   }
 
+
+  bool get supportsDevTool => true;
```

Comment:
What is the purpose of this getter?

Reply:
This was just to show or not the `Riverpod` tab on devtools. But I think I might have a better way of doing that. We could just use a try-catch instead. You can see the implementation here https://github.com/flutter/devtools/pull/4210/files#diff-90638f67ff3222b19bb967f40f76a108841c7a1b6917d8f853d8a04c55d0bb9eR9 .

Reply:
What about making this the package version?
We'll want to expose the version anyway to know if a feature newly added devtool features are supported or not in the future.

This would of course need a unit test to check that the version is in sync with the pubspec.yaml

Reply:
yes, I tried adding the version, but couldn't find a consistent way of getting if from the pubspec. But now that you mentioned the test we could just have a test that reads the pubspec and compare the versions. 🤔 

Is this OK for you? We will need to also add some documentation for it, I suppose.

Reply:
I was thinking here, instead of using the actual package version, why not create a "virtual" versioning for devtools support? Like we could call something like "devtoolsSupportVersion" and start it at `1`, then we just increment the number every time we add a new feature. This way we don't need to always change the version when `riverpod` version changes, we would only need to change when we change the devtool implementation. 

WDYT?